client/gutenberg/extensions/subscriptions/editor.js

```diff
@@ -0,0 +1,60 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+//import './editor.scss';
+import edit from './edit';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import JetpackBlockType from 'gutenberg/extensions/presets/jetpack/utils/jetpack-block-type';
+
+const SubscriptionBlock = new JetpackBlockType( 'subscriptions', {
+	title: __( 'Subscriptions' ),
+
+	description: (
+		<Fragment>
+			<p>{ __( 'Invite visitors to subscribe to your site.' ) }</p>
+		</Fragment>
+	),
+
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 208 128">
```

Comment:
We should be using Gutenberg's svg components (all of these should be replaced):

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/primitives/svg/index.js